### ISSUE TITLE
no-misused-spread rule has a badly-formatted message

### ISSUE THREAD

**Issue by connorshea** [oxc-project/tsgolint#595]
https://github.com/oxc-project/tsgolint/blame/469236f222f1a090d8c8733a4d4a5ca39eb069db/internal/rules/no_misused_spread/no_misused_spread.go#L60-L68

This is not a completely new problem, I tried it with oxlint 1.36.0 and tsgolint 0.10.0 and it happens the same way.

Config:

```jsonc
{
  "plugins": ["typescript"],
  "categories": { "correctness": "off" },
  "rules": {
    "typescript/no-misused-spread": "error",
  }
}
```

File:

```ts
declare const typeAwareViolation: string;

const typeAwareViolationString = [...typeAwareViolation];

console.log(typeAwareViolation);
```

Result:

```
pnpm oxlint --type-aware

  × typescript-eslint(no-misused-spread): Using the spread operator on a string can mishandle special characters, as can `.split("")`.
  │ '- `...` produces Unicode code points, which will decompose complex emojis into individual emojis- .split("") produces UTF-16 code units, which breaks rich characters in many languagesConsider using `Intl.Segmenter` for locale-aware string decomposition.Otherwise, if you don't need to preserve emojis or other non-Ascii characters, disable this lint rule on this line or configure the 'allow'
  │ rule option.
   ╭─[src/index.ts:3:35]
 2 │ 
 3 │ const typeAwareViolationString = [...typeAwareViolation];
   ·                                   ─────────────────────
 4 │ 
   ╰────

Found 0 warnings and 1 error.
Finished in 156ms on 3 files with 1 rules using 8 threads.
```

The linebreak (and/or lack thereof) seems to screw things up. We should probably move part of this to the help message field.
